packages/actor-query-operation-leftjoin/README.md

```diff
@@ -41,3 +41,4 @@ After installing, this package can be added to your engine's configuration as fo
 
 * `mediatorQueryOperation`: A mediator over the [Query Operation bus](https://github.com/comunica/comunica/tree/master/packages/bus-query-operation).
 * `mediatorJoin`: A mediator over the [RDF Join bus](https://github.com/comunica/comunica/tree/master/packages/bus-rdf-join).
+* `expressionEvaluatorFactory`: A factory to create an [Expression Evaluator](https://github.com/comunica/comunica/tree/master/packages/expression-evaluator);
```

Comment:
Could you also add this param to the example above in this file? (also applies to the other readme's using expressions)